diff --git a/charts/minio/templates/_helper_create_buckets.sh b/charts/minio/templates/_helper_create_buckets.sh
index 3cabb5f1f..17f408259 100644
--- a/charts/minio/templates/_helper_create_buckets.sh
+++ b/charts/minio/templates/_helper_create_buckets.sh
@@ -62,7 +62,7 @@ createBucket() {
   # At this point, the bucket should exist, skip checking for existance
   # Set policy on the bucket
   echo "Setting policy of bucket '$BUCKET' to '$POLICY'."
-  /usr/bin/mc policy $POLICY myminio/$BUCKET
+  /usr/bin/mc policy set $POLICY myminio/$BUCKET
 }
 
 connectToMinio
